libbeat/docs/monitoring/monitoring-internal-collection.asciidoc

```diff
@@ -11,22 +11,36 @@
 
 [role="xpack"]
 [[monitoring-internal-collection]]
+
+
+[IMPORTANT]
+=========================
+Metricbeat is the recommended method for collecting and shipping monitoring data
+to a monitoring cluster.
+
+If you have previously configured internal collection, you should migrate to
+using Metricbeat collection. One should either use Metricbeat collection or
```

Comment:
```suggestion
using {metricbeat} collection. One should either use {metricbeat} collection or
```

Reply:
Using "should" makes this sound optional. I'd suggest being more direct and replacing the sentence that begins with "One should..." with something like: 

```
If you use {metricbeat} collection, make sure internal collection is disabled. You cannot use both {metricbeat} and internal collection.
```